### ISSUE TITLE
Can't commit due to eslint bug

### ISSUE THREAD

**Issue by pauliusguzas** [flexn-io/graybox#7]
Reproduce:
1. do` yarn bootstrap-clean`
2. do `git commit -a -m "update"`

```
pauliusguzas@Pauliuss-MacBook-Pro graybox % git commit -a -m "update"
husky > pre-commit (node v16.14.2)
$ yarn compile && yarn lint && yarn test
$ npx lerna run compile
lerna notice cli v6.6.2

> @flexn/graybox:compile

$ tsc -b tsconfig.json

 ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Successfully ran target compile for project @flexn/graybox (1s)

$ npx eslint ./packages

Oops! Something went wrong! :(

ESLint: 8.40.0

ESLint couldn't find the plugin "@typescript-eslint/eslint-plugin".

(The package "@typescript-eslint/eslint-plugin" was not found when loaded as a Node module from the directory "/Users/pauliusguzas/Desktop/graybox".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install @typescript-eslint/eslint-plugin@latest --save-dev

The plugin "@typescript-eslint/eslint-plugin" was referenced from the config file in ".eslintrc.js » @flexn/eslint-config".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky > pre-commit hook failed (add --no-verify to bypass)
pauliusguzas@Pauliuss-MacBook-Pro graybox %
```

**Comment by pavjacko** [comment]
@pauliusguzas fixed

**Comment by pauliusguzas** [comment]
```
pauliusguzas@Pauliuss-MacBook-Pro graybox % git commit -a -m "update"
husky > pre-commit (node v16.14.2)
$ yarn compile && yarn lint && yarn test
$ npx lerna run compile
lerna notice cli v6.6.2

> @flexn/graybox:compile

$ tsc -b tsconfig.json

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Successfully ran target compile for project @flexn/graybox (1s)

$ npx eslint ./packages
$ npx jest --coverage
● Validation Error:

  Preset react-native not found.

  Configuration Documentation:
  https://jestjs.io/docs/configuration

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky > pre-commit hook failed (add --no-verify to bypass)
pauliusguzas@Pauliuss-MacBook-Pro graybox %
```
@pavjacko still getting error, though different this time

**Comment by pauliusguzas** [comment]
fixed

**Comment by mihaiblaga89** [comment]
released in 0.21.0
